src/style/app.module.css

```diff
@@ -6390,3 +6487,582 @@ button[data-testid='createPostBtn'] {
+}
+.profileDropdown .dropdown-toggle .btn .btn-normal {
+  display: none !important;
+  background-color: transparent !important;
+}
+.dropdownToggle {
+  background-image: url(../../../public/images/svg/angleRight.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-color: var(--profile-bg) !important;
+}
+
+.dropdownToggle::after {
+  border-top: none !important;
+  border-bottom: none !important;
+}
+.avatarStyle {
+  border-radius: 100%;
+}
+
+/* SuperAdminScreen */
+
+.pageContainer {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: var(--profile-bg) !important;
+  padding: 1rem 1.5rem 0 calc(300px + 2rem + 1.5rem);
+}
+
+.expand {
+  padding-left: 4rem;
+  animation: moveLeft 0.5s ease-in-out;
+}
+
+.contract {
+  padding-left: calc(300px + 2rem + 1.5rem);
+  animation: moveRight 0.5s ease-in-out;
+}
+
+.collapseSidebarButton {
+  --bs-btn-active-color: var(--profile-bg);
+  --bs-btn-active-bg: var(--profile-bg);
+  --bs-btn-active-border-color: var(--profile-bg);
+  position: fixed;
+  height: 40px;
+  bottom: 0;
+  z-index: 9999;
+  width: calc(300px + 2rem);
+  background-color: rgba(245, 245, 245, 0.7);
+  color: black;
+  border: none;
+  border-radius: 0px;
+}
+
+.collapseSidebarButton:hover,
+.opendrawer:hover {
+  opacity: 1;
+  background-color: var(--profile-bg);
+  box-shadow: var(--hover-shadow);
+  color: black !important;
+}
+.opendrawer {
+  --bs-btn-active-color: var(--profile-bg);
+  --bs-btn-active-bg: var(--profile-bg);
+  --bs-btn-active-border-color: var(--profile-bg);
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  width: 40px;
+  height: 100vh;
+  z-index: 9999;
+  background-color: var(--profile-bg);
+  border: none;
+  border-radius: 0px;
+  margin-right: 20px;
+  color: black;
+}
+
+@media (max-width: 1120px) {
+  .contract {
+    padding-left: calc(250px + 2rem + 1.5rem);
+  }
+  .collapseSidebarButton {
+    width: calc(250px + 2rem);
+  }
+}
+
+/* For tablets */
+@media (max-width: 820px) {
+  .pageContainer {
+    padding-left: 2.5rem;
+  }
+
+  .opendrawer {
+    width: 25px;
+  }
+
+  .contract,
+  .expand {
+    animation: none;
+  }
+
+  .collapseSidebarButton {
+    width: 100%;
+    left: 0;
+    right: 0;
+  }
+}
+
+@keyframes moveLeft {
+  from {
+    padding-left: calc(300px + 2rem + 1.5rem);
+  }
+
+  to {
+    padding-left: 1.5rem;
+  }
+}
+
+@keyframes moveRight {
+  from {
+    padding-left: 1.5rem;
+  }
+
+  to {
+    padding-left: calc(300px + 2rem + 1.5rem);
+  }
+}
+
+/* OrganizationCard */
+.joinBtn {
+  display: flex;
+  justify-content: space-around;
+  width: 8rem;
+  border-width: medium;
+}
+
+.joinedBtn {
+  display: flex;
+  justify-content: space-around;
+  width: 8rem;
+  color: var(--brown-color);
+  font-weight: bold;
+  background-color: var(--light-blue) !important;
+  border-color: var(--light-blue) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`